generator/src/test/java/org/apache/kafka/message/MessageDataGeneratorTest.java

```diff
@@ -47,7 +47,7 @@ public void testNullDefaults() throws Exception {
                 "      \"nullableVersions\": \"2+\", \"default\": \"null\" }",
                 "  ]",
                 "}")), MessageSpec.class);
-        new MessageDataGenerator().generate(testMessageSpec);
+        new MessageDataGenerator("test").generate(testMessageSpec);
```

Comment:
This should be `org.apache.kafka.common.message` to match the directory that the generated source files are outputted to